tasks/main.yaml

```diff
@@ -50,15 +52,34 @@
   tags:
     - kafka_dirs
 
-- name: Create directory for kafka log files
+- name: Create directory for kafka data log files
   file:
-    path: '{{ kafka_log_dirs }}'
+    path: '{{ kafka_data_log_dirs }}'
     state: directory
     group: '{{ kafka_group }}'
     owner: '{{ kafka_user }}'
   tags:
     - kafka_dirs
 
+- name: Create directory for kafka application logs
+  file:
+    path: '{{ kafka_log_dir }}'
+    state: directory
+    group: '{{ kafka_group }}'
+    owner: '{{ kafka_user }}'
+  tags:
+    - kafka_dirs
+
+- name: Create symlink to kafka application logs
```

Comment:
This task to create a symlink fails with an error for existing Kafka installations as there is already a directory with that name. The Ansible role doesn't explicitly  create the `/opt/kafka/logs` directory, but this will have been created the first time Kafka started due to the below lines in the `/opt/kafka/bin/kafka-run-class.sh` file.

```
  # create logs directory
  if [ ! -d "$LOG_DIR" ]; then
    mkdir -p "$LOG_DIR"
  fi
```